prow/plugins.yaml

```diff
@@ -29,6 +34,9 @@ config_updater:
       namespace: boskos
     prow/cluster/jobs/**/*.yaml:
       name: job-config
+    label_sync/labels.yaml:
```

Comment:
Do we want to include the label sync change in this PR?

Reply:
Whoops, nope.